src/getImports/packageJsonParser.js

```diff
@@ -0,0 +1,37 @@
+export function getPackages(fileName, source) {
+  const packages = [];
+
+  const lines = [];
+  source.split(/\r?\n/).forEach(function(line) {
+    lines.push(line);
+  });
+
+  const pjson = JSON.parse(source);
+
+  for (var dep in pjson.dependencies) {
+    var p = {};
+    p.fileName = fileName;
+    p.name = dep;
+    p.loc = findLoc(dep, lines);
+    p.line = p.loc.start.line;
```

Comment:
I think we're missing the `version` property. Which looks is being returned by the `htmlParser`:
https://github.com/snyk/vulncost/blob/77dc018fb3282cbac05486389ef0a8855ffc643e/src/getImports/htmlParser.js#L91

Reply:
Check the js / ts parser. By default we take the latest version if no manufest is available or else we check the version in the manifest.

For html when using a cdn we do not depend on a manifest file but the version inline.